packages/node_modules/@ciscospark/widget-roster/src/components/add-participant/index.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import classNames from 'classnames';
+import {
+  ICON_TYPE_EXIT
+} from '@ciscospark/react-component-icon';
+import Button from '@ciscospark/react-component-button';
+import Spinner from '@ciscospark/react-component-spinner';
+import PersonListItem from '@ciscospark/react-component-person-list-item';
+
+import styles from './styles.css';
+
+export default function AddParticipant(props) {
+  const baseClassPrefix = `ciscospark-roster-add-participant`;
+
+  function handleChange(e) {
+    props.onChange(e.target.value);
+  }
+
+  function handleClose() {
+    props.onChange(``);
+    props.onDismiss();
+  }
+
+  function handleClick(person) {
+    props.onAddPerson(person);
+  }
+
+  let results;
+  if (props.searchResults) {
+    if (props.searchResults.inProgress) {
+      // eslint-disable-next-line no-extra-parens
+      results = (
+        <div className={classNames(`${baseClassPrefix}-results-loading`, styles.resultsLoading)}>
+          <Spinner />
+        </div>
+      );
+    }
+    else if (props.searchResults.results && props.searchResults.results.length) {
+      // eslint-disable-next-line arrow-body-style
+      const listing = props.searchResults.results.map((person) => {
+        const onClick = handleClick.bind(this, person);
+        return (
+          // eslint-disable-next-line jsx-a11y/no-static-element-interactions
+          <div
+            key={person.id}
+            onClick={onClick}
+            onKeyPress={onClick}
+            role="button"
+            tabIndex="0"
+          >
+            <PersonListItem
+              person={person}
+            />
+          </div>
+        );
+      });
+      // eslint-disable-next-line no-extra-parens
+      results = (
+        <div className={classNames(`${baseClassPrefix}-results`, styles.results)}>
+          {listing}
+        </div>
+      );
+    }
+    else if (props.searchTerm) {
+      // eslint-disable-next-line no-extra-parens
+      results = (
+        <div className={classNames(`${baseClassPrefix}-results-none`, styles.resultsNone)}>
+          <div>{props.noResultsMessage}</div>
+        </div>
+      );
+    }
+
```

Comment:
Is there also an empty state we should provide here as well when no searchTerm has been entered?

Reply:
There isn't one for the web client. If we get a design for empty state, we can add it.

Reply:
<img width="293" alt="screen shot 2017-09-25 at 3 12 42 pm" src="https://user-images.githubusercontent.com/99547/30826278-06db776c-a204-11e7-88f2-3efaeefd0447.png">
This is what's in the desktop client.

Reply:
Sorry, this is what's in the roster:
<img width="288" alt="screen shot 2017-09-25 at 3 13 38 pm" src="https://user-images.githubusercontent.com/99547/30826311-2981ed6e-a204-11e7-95b7-c943b7824021.png">

Reply:
Yea, we don't have that asset, nor does the web client display it.